brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
So let's confirm that. If it is only IE8, I am all in for dropping it. We
then keep the X domain stuff to support IE9 and people can simply not
upgrade it.

On Thursday, November 26, 2015, Eric Meadows-Jönsson <
notifications@github.com> wrote:

> In brunch-config.js
> <https://github.com/phoenixframework/phoenix/pull/1374#discussion_r46008867>
> :
>
> > @@ -31,8 +31,7 @@ exports.config = {
> >    plugins: {
> >      babel: {
> >        // Do not use ES6 compiler in vendor code
> > -      ignore: [/^(web\/static\/vendor)/],
> > -      loose: "all"
>
> IIRC loose mode is to support IE8, XDomainRequest is to support IE8-9.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phoenixframework/phoenix/pull/1374/files#r46008867>.
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*